go/saltpackKeyHelpers/saltpackHelperMocks/saltpack_mock_pseudonym_resolver.go

```diff
@@ -0,0 +1,48 @@
+package saltpackHelperMocks
```

Comment:
Can you change the directory from `go/saltpackKeyHelpers` to something shorter `go/saltpackkeys` maybe?  Plus, I don't think directories are supposed to be camel cased.

Reply:
Done.